crates/api_models/src/gsm.rs

```diff
@@ -141,4 +206,8 @@ pub struct GsmResponse {
     pub unified_code: Option<String>,
     /// error message unified across the connectors
     pub unified_message: Option<String>,
+    /// category in which error belongs to
+    pub error_category: Option<String>,
```

Comment:
api-reference should be showing all the possible values in the response too, better to make this an enum